.cirrus.yml

```
@@ -67,15 +67,21 @@ skip_task_on_pr: &SKIP_TASK_ON_PR
   # Skip this task on PRs if it does not have the CI: Full label,
   # it continues to run for direct pushes to master/release.
   skip: >
-    ! ( $CIRRUS_PR == '' || $CIRRUS_PR_LABELS =~ '.*CI: Full.*' )
+    ( ! ($CIRRUS_PR == '' || $CIRRUS_PR_LABELS =~ '.*CI: Full.*') || $CIRRUS_PR_LABELS =~ '.*CI: Skip All' )
+
+skip_cluster_test_on_pr: &SKIP_CLUSTER_TEST_ON_PR
+  # Skip this task on PRs if it does not have the CI: Full label,
+  # it continues to run for direct pushes to master/release.
+  skip: >
+    ( ! ($CIRRUS_PR == '' || $CIRRUS_PR_LABELS =~ '.*CI: (Full|Cluster Test).*') || $CIRRUS_PR_LABELS =~ '.*CI: Skip All' )
 
 zam_skip_task_on_pr: &ZAM_SKIP_TASK_ON_PR
   # Skip this task on PRs unless it has the "CI: Full" or "CI: ZAM" label
   # or files in src/script_opt/** were modified.
   # It continues to run for direct pushes to master/release, as
   # CIRRUS_PR will be empty.
   skip: >
-    ! ( $CIRRUS_PR == '' || $CIRRUS_PR_LABELS =~ '.*CI: (Full|ZAM).*' || changesInclude('src/script_opt/**') )
+    ( ! ($CIRRUS_PR == '' || $CIRRUS_PR_LABELS =~ '.*CI: (Full|ZAM).*' || changesInclude('src/script_opt/**')) || $CIRRUS_PR_LABELS =~ '.*CI: Skip All' )
```

```suggestion
    ( ! ($CIRRUS_PR == '' || $CIRRUS_PR_LABELS =~ '.*CI: (Full|ZAM).*' || changesInclude('src/script_opt/**')) ) || $CIRRUS_PR_LABELS =~ '.*CI: Skip All' 
```

Wonder if this would make it clearer, but not a big improvement, I'm always confused staring at these conditions.